src/components/Layout/Layout.scss

```diff
@@ -1,5 +1,6 @@
 $secondary-basis: layout-width(secondary, min);
 $primary-basis: layout-width(primary, min);
+$even-width-basis: layout-width(page-content, partially-condensed);
```

Comment:
This is the exact width that I want, am I using this function correctly though? I'm not sure what `page-content` and `partially-condensed` represent.

Reply:
Ah, yeah. That is the width for the breakpoint where we start to condense the padding on the page. I would add a new value to the map for `equal-width` here (even if it is the same value): https://github.com/Shopify/polaris-react/blob/master/src/styles/foundation/layout.scss